README.md

```diff
@@ -7,67 +7,55 @@ Meson is a mixnet for cryptocurrency transactions. Meson is based on the [Katzen
 ## Docs
 These are the basics for joining the Meson mixnet as a provider, authority or mix. For more in-depth documentation on the Meson project, visit out [project site](hashcloak.com/Meson).
 
-### How to run a provider
+### How to run a provider or mix node
 
 Requirements:
 - `go` version 1.13
 - `docker swarm`
 
-All of our infrastructure uses solely docker setups. You will first need to generate a provider config and its PKI keys. The easiest way to do that is by using our [genconfig](https://github.com/hashcloak/genconfig/) script:
+All of our infrastructure uses docker setups. You will first need to generate a provider config and its PKI keys. The easiest way to do that is by using our [genconfig](https://github.com/hashcloak/genconfig/#genconfig) script:
 
 ```bash
-$ git clone https://github.com/hashcloak/genconfig.git
-$ cd genconfig
-$ go run main.go -a 138.197.57.19  \ # This is the current authority config address
-  -prov \ # prov indicates to generate a provider config
-  -name <provider-name> \ # Provider name
-  -pubtcp4 <your public ip address> # public ipv4 address
+git clone https://github.com/hashcloak/genconfig.git
+cd genconfig
+go run main.go \
+  -a 138.197.57.19 \ # current ip address of authority
+  -authID RJWGWCjof2GLLhekd6KsvN+LvHq9sxgcpra/J59/X8A= \ # current public key of authority
+  -name provider-name \ # your provider name
+  -ipv4 1.1.1.1 \ # your public ipv4 address
+  -provider \ # flag to indicate you only want a provider config
 ```
 
-The last command will make a directory called `output` with another subdir called `<provider-name>`. Send us your public at our [Riot.im](https://riot.im/app/#/room/#hashcloak:matrix.org) room. We will then help you to get added as a provider.
+This will make a directory called `./output/provider-name`. Send us your public key at our [Riot.im](https://riot.im/app/#/room/#hashcloak:matrix.org) room. We will then help you to get added as a provider. Once you give is your public key you can get your node running with:
```

Comment:
Our riot room is private. Maybe a better way to send public keys might be instead email. For now, instruct the user to send their public keys to `info@hashcloak.com`